tests/main/snaps-state/task.yaml

```diff
@@ -122,6 +122,12 @@ execute: |
         echo "$sum" | awk '{print $1}'
     }
 
+    # Skip test repack when the sru validation is being executed, in this
+    # scenario the deb package used comes from the sru and repack is not done
+    if [ "$SRU_VALIDATION" = 1 ]; then
```

Comment:
I think it would be fine to even skip the entire test, given that it's only about the snapd-state test tool. Testing this tool as part of the SRU seems not very important